### PR TITLE
feat: add workflow resolver

### PR DESCRIPTION
**Pull request by darkskygit**
fix AFF-1166

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#7311** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7311?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7231** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7231?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7159** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7159?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7123** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7123?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7119** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7119?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/server/src/data/migrations/utils/prompts.ts [hunk 1]
```diff
@@ -492,6 +492,12 @@ content: {{content}}`,
       },
     ],
   },
+  {
+    name: 'Create a presentation:workflow',
+    action: 'Create a presentation:workflow',
+    model: 'Create a presentation',
```
Comment (forehalo) [inline packages/backend/server/src/data/migrations/utils/prompts.ts]: model what?

Comment (darkskygit) [reply packages/backend/server/src/data/migrations/utils/prompts.ts]: used for workflow, in workflow api we will use model name to find workflow graph

Comment (darkskygit) [reply packages/backend/server/src/data/migrations/utils/prompts.ts]: i. added a prefix `workflow:` to make it clearly

**Review by forehalo — APPROVED** [review]

**Pull request merged**